### ISSUE TITLE
Data Wrangling Info & Chat

### ISSUE THREAD

**Issue by wenyunie** [wenyunie/speed_dating_analysis#7]
| Attribute Code | Timing | Type | Notes |
| ------------- | ------------- | ------------- | ------------- |
| attr1_1  | signup/Time1 | Self Attribute Importance Rating |  |
| attr1_2  | followup/Time2 |  Self Attribute Importance Rating | Different Scales in different waves |
| attr1_3  | followup2/ Time3 |  Self Attribute Importance Rating |  |
| attr2_1  | signup/Time1 |  Perceived Opposite Sex Attribute Importance Rating |  |
| attr2_2  | followup/Time2 |  Perceived Opposite Sex Attribute Importance Rating | Total Point Assignment |
| attr2_3  | followup2/Time3 |  Perceived Opposite Sex Attribute Importance Rating |  |
| attr3_1  | signup/Time1 |  Self Rating |  |
| attr3_2  | followup/Time2 |  Self Rating |  |
| attr4_1  | signup/Time1 |  Perceived Fellow Attribute Importance Rating | Different Scales in different waves |
| attr4_2  | followup/Time2 |  Perceived Fellow Attribute Importance Rating | Total Point Assignment |
| attr4_3  | followup2/Time3 |  Perceived Fellow Attribute Importance Rating |  |
| attr5_1  | signup/Time1 |  Perceived Other Rating On Self |  |
| attr5_2 | followup/Time2 |  Perceived Other Rating On Self |  |
| attr5_3 | followup2/Time3 |  Perceived Other Rating On Self |  |
| attr7_2 | followup/Time2 |  Self Attribute Importance Rating with reflection | Total Point Assignment |
| attr7_2 | followup2/Time3 |  Self Attribute Importance Rating with reflection | Total Point Assignment |

**Comment by wenyunie** [comment]
This dataset is far more fun than I thought it would be.

**Comment by monazhu** [comment]
Yeah it's actually quite a rich data set! This could quite literally be someone's PhD dissertation 😂

**Comment by monazhu** [comment]
@wenyunie, I was looking at the analysis file more closely and I realized we might have used the wrong data column for the analysis (because we would actually expect there to be a correlation between self vs other perceived attractiveness, even if we tend to over-rate our own attractiveness). I think rather than summarizing over the `attr` column (which is how their partners rated them), we should be using `attr_o` (how they were rated by their partners. 

I quickly looked at the stats and graphs and these results were more in-line with what I would expect. I'll send along the output file I did quickly via slack so you can see what I mean

**Comment by rorywhite200** [comment]
Great spot Mona! Thanks. On a tangential note, there was an interesting paper that talks about the weak correlation between self vs other perceived attractiveness and said something like if everyone had the same bias then it would be a perfect correlation. Apparently unattractive people completely overestimate their attractiveness while attractive people underestimate, which is fascinating. 

![SJOP-61-471-g001](https://github.com/wenyunie/speed_dating_analysis/assets/125914446/2d6c71fb-aee9-4d52-88d8-94f8c277b8f9)

**Comment by wenyunie** [comment]
@rorywhite200 This could be a good explanation of the weak correlation! Is this plot from our dataset or the study in the link? Please feel free to change anything about the data analysis part if you would like to include this plot/analysis in our report.

**Comment by wenyunie** [comment]
I saw the plot above is from the linked study, and just out of curiosity, I did the same thing with our dataset. The general trend holds, only that attractive people are accurate about themselves. 
<img width="660" alt="image" src="https://github.com/wenyunie/speed_dating_analysis/assets/143786716/8f86b85c-a771-46fa-8ad4-fa15da4fc881">

**Comment by wenyunie** [comment]
It seems pretty close to their study2 result:
<img width="791" alt="image" src="https://github.com/wenyunie/speed_dating_analysis/assets/143786716/a96dcaa0-c809-4053-9486-a55c04a268d2">

**Comment by rorywhite200** [comment]
Wow that is strikingly similar! Cool work! I wonder if it is too late to include this? I guess it goes a bit beyond our hypothesis. Maybe in one of the future weeks we will get an opportunity to expand our analysis

**Comment by monazhu** [comment]
Ouu this is super neat stuff! I really love that we're all digging deep into the data and finding cool gems 😄 I do agree with Rory though that a quantile analysis is beyond the scope of milestone 1. It would definitely be neat to have in the final-final-final report though, if we have the opportunity to include it since you did all this work!

**Comment by monazhu** [comment]
Also this is a total random aside, but a correlation of ~0.3 in social sciences is honestly considered quite big (especially since it's others' rating vs your own rating). If I saw a correlation of like 0.6+ in a social sciences paper I actually might start to get worried 😂
